# Generated by Django 3.1.4 on 2021-01-07 22:15

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='Perfil',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('imagen', models.ImageField(blank=True, null=True, upload_to='usuarios/images')),
                ('creado', models.DateTimeField(auto_now_add=True)),
                ('nacim', models.DateField()),
                ('num_celular', models.BigIntegerField(blank=True, null=True)),
                ('pais', models.CharField(max_length=50)),
                ('biografia', models.TextField(blank=True, null=True)),
                ('sentecia', models.DurationField(blank=True, null=True)),
                ('nick', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.CreateModel(
            name='BannData',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('codigo_foro', models.IntegerField()),
                ('titulo', models.CharField(max_length=70)),
                ('causa', models.CharField(max_length=100)),
                ('nick', models.ManyToManyField(to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
